src/userobjects/MoabSkinner.C

```diff
@@ -6,6 +6,8 @@
 #include "BinUtility.h"
 #include "GeometryUtils.h"
 #include "UserErrorChecking.h"
+#include "DisplacedProblem.h"
+#include "FEProblemBase.h"
```

Comment:
Hmm ok I think we may have a problem here -- this is going to cause some circular logic, because we're later wanting to `#include` the `MoabSkinner` inside of a class which derives from `FEProblemBase`. 

How was this working for you in your other branch where you are modifying the `OpenMCCellAverageProblem` as well?